### ISSUE TITLE
NTR: Healthcare activities just before delivery

### ISSUE THREAD

**Issue by SheenaTomko** [VEuPathDB-ontology/VEuPathDB-ontology#439]
For new term requests, please provide the following information:
**Term Label** [required]
Healthcare activities just before delivery

**Synonyms** [optional]

**Definition** [required except for organism name]
Multifilter to gather variables about activities that are on the birth checklist and supposed to be performed just before delivery

**Suggested Parent Term** [required]
Please look in the ontology hierarchy in a browser such as [OLS](http://www.ebi.ac.uk/ols/ontologies/eupath) or choose one from following list: 
- [ ] Material (independent of other entities and can be maintained through time, such as person, cell, sample)
- [ ] Process (has temporal proper parts and for some time depends on some material entity, such as life cycle, cell division, heart beating, sleeping)
- [ ] Information (generically dependent on some artifact and stands in relation of aboutness to some entity, such as data, identifier, document)
- [ ] Characteristic/Quality (inheres in or is borne by other entities, such as height, color, disease)

**Examples** [optional]

**Contributor** [optional]
If you would like to be a contributor of the term or anyone you think that contributes to the term, such as label, definition, please provide the name and ORCID ID.

**Requester** [optional]
Sheena/ClinEpiDB

**Comment by johnwjudkins** [comment]
EUPATH:0061765 information on health care immediately preceding baby delivery
An information content entity that is about other health care encounters that occur immediately before baby delivery and involve the same patient.

**Comment by SheenaTomko** [comment]
Looks good to me

**Comment by johnwjudkins** [comment]
Changed to plan specification to match similar terms.

**Comment by johnwjudkins** [comment]
EUPATH:0061765
label: information on health care immediately preceding baby delivery
definition: An information content entity that is about other health care processes that occur immediately before baby delivery and involve the same patient.
parent: information content entity

Changed back because this seems more like a record (information) rather than a plan specification. Does this make sense @SheenaTomko?

**Comment by SheenaTomko** [comment]
Yes, this isn't about a plan, it's about what ended up happening right before delivery started

**Comment by johnwjudkins** [comment]
@zhengj2007 please review

**Comment by zhengj2007** [comment]
Better remove 'other' in the definition.
